analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6-OSS-disclosure.txt

```diff
@@ -0,0 +1,1296 @@
+
+This General Public License does not permit incorporating your program into
+proprietary programs. If your program is a subroutine library, you may consider
+it more useful to permit linking proprietary applications with the library. If
+this is what you want to do, use the GNU Lesser General Public License instead
+of this License.
+
+License text of the Autoconf-exception-generic as an exception to the
+GPL-2.0-or-later:
+
+As a special exception to the GNU General Public License, if you distribute this
+file as part of a program that contains a configuration script generated by
+Autoconf, you may include it under the same distribution terms that you use for
+the rest of that program.
+
+--------------------------------------------------------------------------------
+
+Gettext-Permission-Notice
+
+This file is free software; the Free Software Foundation
+gives unlimited permission to copy and/or distribute it,
+with or without modifications, as long as this notice is preserved.
+
+This file can be used in projects which are not available under
+the GNU General Public License or the GNU Library General Public
+License but which still want to provide support for the GNU gettext
+functionality.
+Please note that the actual code of the GNU gettext library is covered
+by the GNU Library General Public License, and the rest of the GNU
+gettext package is covered by the GNU General Public License.
+They are *not* in the public domain.
+
+--------------------------------------------------------------------------------
+
+License-of-GNU-Licenses
+
+Everyone is permitted to copy and distribute verbatim copies
+of this license document, but changing it is not allowed.
+
+--------------------------------------------------------------------------------
+
+FSFULLRWD
+
+This Makefile.in is free software; the Free Software Foundation gives unlimited
+permission to copy and/or distribute it, with or without modifications, as long
+as this notice is preserved.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY, to the extent permitted by law; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+--------------------------------------------------------------------------------
+
+FSFULLR
+
+This file is free software; the Free Software Foundation gives unlimited
+permission to copy and/or distribute it, with or without modifications, as long
+as this notice is preserved.
+
+--------------------------------------------------------------------------------
+
+FSFUL
+
+This file, Rules-quot, and its auxiliary files (listed under
+DISTFILES.common.extra1) are free software; the Free Software Foundation
+gives unlimited permission to use, copy, distribute, and modify them.
+
+--------------------------------------------------------------------------------
+
+FSFUL
+
+This configure script is free software; the Free Software Foundation gives
```

Comment:
please add additionally:
This config.status script is free software; the Free Software Foundation
gives unlimited permission to copy, distribute and modify it.
See file "configure"